### PR TITLE
setup react and added linters

### PR DESCRIPTION
**Pull request by MAleemH**
I have:

- Added linters
- Setup React

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @MAleemH, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the README file uses the default React Template Readme. Please use the [Microverse template](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,262 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (TadesseDev) [inline README.md]: - [ ] Notice the link for your MIT file is broken, Please make sure to add the MIT license file and update its content in regard to your name and project. Also replace the `[YOUR NAME]` part with your name, and `[APP TYPE]` with your app type.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @MAleemH 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x]  Correct GitFlow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @MAleemH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**